framework/src/org/checkerframework/framework/type/AnnotatedTypeFactory.java

```diff
@@ -136,8 +135,9 @@
  *
  * <p>Type system checker writers may need to subclass this class, to add implicit and default
  * qualifiers according to the type system semantics. Subclasses should especially override {@link
- * AnnotatedTypeFactory#addComputedTypeAnnotations(Element, AnnotatedTypeMirror)} and {@link
- * #addComputedTypeAnnotations(Tree, AnnotatedTypeMirror)}.
+ * AnnotatedTypeFactory#addComputedTypeAnnotations(Element, AnnotatedTypeMirror)}, {@link
+ * #addComputedTypeAnnotations(Tree, AnnotatedTypeMirror)}, and {@link
+ * #addDefaultAnnotations(AnnotatedTypeMirror)}.
```

Comment:
I would list the third one as a temporary hack, not as one of the intended extension points.